runner/runner.go

```diff
@@ -21,7 +21,7 @@ type Command struct {
 	Timeout time.Duration
 
 	// Runner can optionally use this to run against a particular snapshot. Empty value is ignored.
-	SnapshotId string
+	SnapshotID string
```

Comment:
Nit: Is changing the capitalization necessary?  Is this going to cause a bunch of breaking changes in twitter source?